api/src/pcapi/core/finance/api.py

```diff
@@ -635,9 +635,19 @@ def price_booking(
 
         _delete_dependent_pricings(booking, "Deleted pricings priced too early")
 
-        pricing = _price_booking(booking)
+        event, pricing = _price_booking(booking)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏on pourrait appeler la variable `finance_event` comme son type

Je connais mal ce code là certes, mais en lisant event j'ai d'abord cru que c'est que le booling concerne un évènement 
finance_event ne laisse aucun doute